### ISSUE TITLE
atalk does not remember changed password

### ISSUE THREAD

**Issue by streaps** [cmeng-git/atalk-android#12]
- change the password of an account on the server
- atalk gets an authentication error
- change the password in atalk to the new password
- atalk connects successfully
- next time atalk starts it tries to login with the old password

**Comment by cmeng-git** [comment]
Thank. I am aware of this problem, and have already fixed the problem. Will be included in next release.

**Comment by streaps** [comment]
I also have problems with setting an additional STUN/TURN server.

**Comment by streaps** [comment]
In 1.4.0 I can now add a stun/turn server, but I cannot change the settings of a stun/turn server or delete one.

**Comment by cmeng-git** [comment]
Just tap on the last saved stun setting. You should see the last saved stun seettings with 3 buttons i.e.Remove, cancel and save. Edit the required parameters and Save. Select Remove to delete the setting.

**Comment by streaps** [comment]
I've already tried that, it doesn't work. When I leave Account Settings and go back to Additional STUN/TURN Servers nothing has changed.

**Comment by cmeng-git** [comment]
Yes. You are right. Look like there are other problems more than I expected. Need to really clean up the code in this area.

**Comment by streaps** [comment]
Works fine in 1.4.1
